juriscraper/pacer/activity_report.py

```diff
@@ -0,0 +1,254 @@
+                except ValueError:
+                    flag = "none"
+                    return flag
+                else:
+                    flag = cols[cols.index('Case Flags:')+3]
+                    return flag
+
+            flags = isFlags(cols)
+
+            if b > 0: # skip first
+                data.append({
+                    u"case":cols[2],
+                    u"case_number":cols[4],
+                    u"docket_report_url":cols[1].xpath('@href')[0],
+                    u"entered": cols[cols.index('Entered: ')+1],
+                    u"time": cols[cols.index('Entered: ')+3],
+                    u"filed": cols[cols.index('Filed: ')+1],
+                    u"closed": cols[8],
+                    u"event": cols[cols.index('Event:')+1],
+                    u"case_flags": flags,
+                    u"category": cols[cols.index('Category:')+1],
+                    u"presider": cols[cols.index('Presider: ')+1],
+                    u"office": cols[cols.index('Office:')+1],
+                    u"docketed_by":cols[cols.index('Document: ')+5],
+                    u"doc1":cols[cols.index('Document: ')+1].xpath('@href')[0],
+                    u"doc_count":cols[cols.index('Document: ')+2]
+
+                })
+
+
+        self._metadata = data
+        return data
+
+    def query(self, open, closed, office, type, start, end, text):
+        """
+         closed = 'on' or ''
+         open = 'on' or ''
+
+         office = 4
+         options:
+        <option value="2">Phoenix Division</option>
+        <option value="3">Prescott Division</option>
+        <option value="4">Tucson Division</option>
+
+
+        type = 'po'
+        options:
+        <select name="case_type" multiple="" size="3">
+        <option selected=""></option>
+        <option value="cv">Civil</option>
+        <option value="cr">Criminal</option>
+        <option value="em">Emergency Matter</option>
+        <option value="mj">Magistrate Judge</option>
+        <option value="mc">Miscellaneous</option>
+        <option value="md">Multi-District Litigation</option>
+        <option value="nr">Non Reportable</option>
+        <option value="nc">Non-Compliance</option>
+        <option value="po">Petty Offense</option>
+        <option value="ph">Pro Hac Vice Case</option>
+        <option value="pt">Probation Transfer</option>
+        <option value="ps">Prose Screening</option>
+        <option value="sc">Sealed Civil</option>
+        <option value="sm">Sealed Miscellaneous</option>
+        <option value="mb">Search Warrant</option>
+        <option value="sa">Staff Attorney Assignment</option>
+        <option value="at">Temporary Case Type</option>
+        <option value="vc">Virtual CR Case for 3rd Judge</option>
+        <option value="wi">Waiver of Indictment</option>
+        <option value="wt">Wire Tap</option>
+        <option value="gj">~Grand Jury</option>
+        </select>
+
+        start = 'm/d/yyyy'
+        end = 'm/d/yyyy'
```

Comment:
These should be date objects. That way calling code doesn't have to think about dumb PACER date formatting decisions.